.gitignore

```diff
@@ -1,5 +1,10 @@
 *.pyc
 *.retry
+*.swp
+
+# collection files
+freeipa-*.tar.gz
```

Comment:
The names for the collections are `freeipa.ansible_freeipa` and `redhat.rhel_idm`.

Reply:
Fixed.